ios/Sources/TimetableFeature/Detail/TimetableDetailScreen.swift

```diff
@@ -23,15 +23,17 @@ public struct TimetableDetailScreen: View {
         ZStack {
             ScrollView {
                 WithViewStore(store) { viewStore in
-                    Text(viewStore.timetable.title.get(by: viewStore.language))
+                    Text(viewStore.timetable.title.get(
+                            by: Foundation.Locale.current.language
```

Comment:
does this really works when change language dynamically?

Reply:
Yes.